groups/group_service.go

```diff
@@ -0,0 +1,30 @@
+package groups
+
+import (
+	"context"
+	"time"
+)
+
+type (
+	// GroupService has ranking methods
+	GroupService interface {
```

Comment:
shouldn't we have a GroupDelete method?

Reply:
We have GroupRemoveAll, which removes everything, including the group itself. Should I change the behaviour to GroupRemoveAll remove all the uids, but not delete and GroupDelete delete it everything?

Reply:
Yes, I guess this would be more clear

Reply:
Done!